drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -74,6 +74,19 @@ static uint32_t get_pllsrc_frequency(size_t pll_id)
 
 static uint32_t get_startup_frequency(void)
 {
+
+	if (IS_ENABLED(STM32_HSI_ENABLED)) {
+		while (LL_RCC_HSI_IsReady() != 1) {
+		/* Wait for HSI ready */
+		}
+	}
+
+	if (IS_ENABLED(STM32_MSIS_ENABLED)) {
+		while (LL_RCC_MSI_IsReady() != 1) {
+		/* Wait for MSI ready */
+		}
+	}
```

Comment:
I understand the problem in resume from low power, but will this work in usual boot ?
This function is called before set_up_fixed_clock_sources which enables these clocks

Reply:
It boots. However this part could still be flagged by `CONFIG_PM` to reduce impact 
Even when exiting the low power mode or booting the same function stm32_clock_control_init is called.